### ISSUE TITLE
NuScenes Boston Split

### ISSUE THREAD

**Issue by MaekTec** [Tsinghua-MARS-Lab/neural_map_prior#16]
Hi,
thanks for the great work!
Do you provide the Boston split that you used in Table.8? Can it be enabled by setting nusc_new_split to True in nuscenes_dataset.py and generated with nusc_split.py?

**Comment by abbyxxn** [comment]
Thank you for your interest in our project. The "new split" is a split newly proposed in the article "[StreamMapNet: Streaming Mapping Network for Vectorized Online HD Map Construction](https://arxiv.org/pdf/2308.12570.pdf)." The new split is a good choice if you want to test the model's performance on datasets without geographical overlap. Yes, it can be enabled by setting nusc_new_split to True within nuscenes_dataset.py and using nusc_split.py for generation. I have tested the effect of the neural map prior on this split, and there has been a notable improvement. I have uploaded the code for generating the Boston split [here](https://github.com/Tsinghua-MARS-Lab/neural_map_prior/tree/main/project/neural_map_prior/data_utils/boston_split_gen). The generation of the Boston split involves a series of calculations, making the use of the new split a better option.
